schemas/dev.amp.snowplow/amp_page_ping/jsonschema/1-0-0

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "dev.amp.snowplow",
```

Comment:
Does `dev` stand for development here? If yes, isn't the plan to have this schema be used in production?

Reply:
It's following the naming convention [originator of data].[owner of schema] that we've used for other schemas. It's dev because AMP's url is on a dev. domain: https://amp.dev

Reply:
Oh I didn't think that they could be on a `.dev` domain, my mistake!